src/workspaces/list/RenderedWorkspaces.tsx

```diff
@@ -185,9 +185,7 @@ const NameCell = (props: CellProps): ReactNode => {
             canAccessWorkspace();
             !!canView && Ajax().Metrics.captureEvent(Events.workspaceOpenFromList, extractWorkspaceDetails(workspace));
           }}
-          tooltip={
-            !canView && 'You do not have access to this workspace. Select the workspace to learn about gaining access.'
-          }
+          tooltip={!canView && 'You do not have access to this workspace. Select the workspace to learn about options.'}
```

Comment:
Minor wording change since the modal may also include the Delete Workspace option.

![image](https://github.com/user-attachments/assets/b22fd0c3-0bed-4e94-afad-0c44ebbacd6a)